packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -86,6 +86,52 @@ const validateAppConfiguration = () => {
     }
 }
 
+/**
+ * Executes a command in the application-extension directory.
+ * @param {string} command - The command to execute.
+ * @param {string} pkgRoot - The root directory of the package.
+ */
+const runCommandInExtensions = (command, pkgRoot) => {
+    const extensionsDir = p.join(pkgRoot, '..', '..', '..', 'app', 'application-extensions')
```

Comment:
why do we need the `.., .., ..`? I wonder if there is a more reliable path resolution.